Libplanet.Extensions.Cocona/Commands/StoreCommand.cs

```diff
@@ -16,6 +17,24 @@ public class StoreCommand
             "The URI denotes the type and path of concrete class for " + nameof(IStore) + "."
             + "<store-type>://<store-path> (e.g., rocksdb+file:///path/to/store)";
 
+        [Command(Description = "List all chain IDs.")]
+        public void ChainIds(
+            [Argument("STORE", Description = StoreArgumentDescription)]
+            string home
+        )
+        {
+            IStore store = Utils.LoadStoreFromUri(home);
```

Comment:
Not important for functionality.

```suggestion
            string storePath
        )
        {
            IStore store = Utils.LoadStoreFromUri(storePath);
```

Reply:
Amended!